doc/runbooks/deployment-failures.md

```diff
@@ -0,0 +1,10 @@
+# Deployment Failures
+
+## Identifying a failure?
+
+- GitHub Actions timeout for a deployment
+
+## How do you resolve it?
+
+- review logs in CloudWatch
+- review logs from CodeDeploy?
```

Comment:
I couldn't find any codedeploy logs. Do you know if they exist?

Reply:
Code Deploy does in fact not have logs, I think the ECS Task should have something, I'll go and figure out where logs might be coming from now